tests/test_modeling_flax_common.py

```diff
@@ -792,7 +792,7 @@ def test_default_params_dtype(self):
             types = flatten_dict(types)
 
             for name, type_ in types.items():
-                self.assertEquals(type_, jnp.float32, msg=f"param {name} is not initialized in fp32.")
+                self.assertEqual(type_, jnp.float32, msg=f"param {name} is not initialized in fp32.")
```

Comment:
Is this change necessary?

Reply:
Not neccesary, I was just getting the following warning `DeprecationWarning: Please use assertEqual instead.` and thought since I was changing to add.
Undid change